src/test/resources/net/sf/jabref/logic/exporter/ModsExportFormatTestMultipleEntries.bib

```diff
@@ -0,0 +1,20 @@
+% Encoding: UTF-8
+
+@Article{,
+  author   = {Shruthi, Basavaradhya Sahukar and Vinodhkumar, Palani and Selvamani},
+  title    = {Proteomics: A new perspective for cancer.},
+  journal  = {Advanced biomedical research},
```

Comment:
I am not aware of the mods fornat, but does it have an issue field`?
Then this would be worth testing, too. Some journals have both volume and issue